[2022-08-23 11:37:46,859] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-10T11:00:00+00:00 [queued]>
[2022-08-23 11:37:46,880] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-10T11:00:00+00:00 [queued]>
[2022-08-23 11:37:46,881] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:37:46,881] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-23 11:37:46,882] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:37:46,907] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): print_world> on 2021-10-10 11:00:00+00:00
[2022-08-23 11:37:46,917] {standard_task_runner.py:52} INFO - Started process 581 to run task
[2022-08-23 11:37:46,922] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'print_world', 'scheduled__2021-10-10T11:00:00+00:00', '--job-id', '1650', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmp5giazxex', '--error-file', '/tmp/tmp7xmh5l7v']
[2022-08-23 11:37:46,923] {standard_task_runner.py:80} INFO - Job 1650: Subtask print_world
[2022-08-23 11:37:47,031] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-10T11:00:00+00:00 [running]> on host 7f48b7aaf0fc
[2022-08-23 11:37:47,196] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=print_world
AIRFLOW_CTX_EXECUTION_DATE=2021-10-10T11:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-10T11:00:00+00:00
[2022-08-23 11:37:47,200] {logging_mixin.py:115} INFO - world
[2022-08-23 11:37:47,201] {python.py:173} INFO - Done. Returned value was: None
[2022-08-23 11:37:47,238] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=print_world, execution_date=20211010T110000, start_date=20220823T113746, end_date=20220823T113747
[2022-08-23 11:37:47,294] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-23 11:37:47,348] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
